instrumentation/okhttp/okhttp-3.0/testing/src/main/groovy/io/opentelemetry/instrumentation/okhttp/v3_0/AbstractOkHttp3Test.groovy

```diff
@@ -45,7 +45,10 @@ abstract class AbstractOkHttp3Test extends HttpClientTest<Request> {
 
   @Override
   int sendRequest(Request request, String method, URI uri, Map<String, String> headers) {
-    return client.newCall(request).execute().code()
+    def response = client.newCall(request).execute()
+    response.body().withCloseable {
+      return response.code()
+    }
```

Comment:
it's really important to close the response body *always* when using okhttp, otherwise we're leaking responses in our test code.

Reply:
😱